com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineMaterialUpgrader.cs

```diff
@@ -416,6 +418,21 @@ public TerrainUpgrader(string oldShaderName)
         }
     }
 
+    public class SpeedTreeUpgrader : MaterialUpgrader
```

Comment:
Make it private or internal (we made the other ones public by mistake)

Reply:
> 
> 
> LGTM. Minor questions and also lacking changelog entry.

Did I add the changelong in the wrong place? https://github.com/Unity-Technologies/Graphics/pull/1350/commits/50f7554d02a6ed2d65b6d0f72f6bb695ef3963d1

Reply:
Made internal in https://github.com/Unity-Technologies/Graphics/pull/1350/commits/cae0ba0b7a3c48d28f80f74322ba22707a9581c2